iocage_lib/ioc_json.py

```diff
@@ -2200,6 +2235,20 @@ def json_check_prop(self, key, value, conf):
         elif key in props.keys():
             # Either it contains what we expect, or it's a string.
 
+            if DEPRECATED in props[key]:
+                iocage_lib.ioc_common.logit(
+                    {
+                        'level': 'EXCEPTION',
+                        'message': f'The `{key}` parameter is deprecated, is '
```

Comment:
*`and is no longer`

Reply:
Just because grammar or is there a change in meaning here? There's a comma-and on the next line because I'm listing three things: deprecation, ignoring, inability to set.

Tone: not taking any offense at a grammar correction here, just trying to understand the change I need to make.